source/test.ts

```diff
@@ -23,7 +23,7 @@ test('auto removal on initial Map', async t => {
 
 	t.true(map.has('unicorn'));
 
-	await delay(600);
+	await delay(605);
```

Comment:
Any reason you changed this to 605? :)

Reply:
It felt like 600 was “just enough” so I thought “a little more than just enough” would be more likely to trigger the issue, if it happened.

Reply:
Aha I see, fair point!